### PR TITLE
Updates to startup: Update commander version, add debug functionality…

### PR DESCRIPTION
**Pull request by marybeshaw**
…, add ability to pass arguments through a Procfile.

### CODE REVIEW

File: bin/startup-start [hunk 1]
```diff
@@ -15,11 +15,21 @@ var dirname = require('path').dirname;
 var resolve = require('path').resolve;
 var exists = fs.existsSync;
 
+var debug = function nopDebug(msg){};
+
+if (process.env.DEBUG_STARTUP === 'true') {
+  debug = function debug(msg) {
+    console.log(msg);
+  };
```
Comment (redbugz) [inline bin/startup-start]: I don't understand this code. Can you explain what you are trying to accomplish?

Comment (marybeshaw) [reply bin/startup-start]: This is something @gechols added. I think it adds logging of startup errors.

Comment (marybeshaw) [reply bin/startup-start]: https://github.com/marybeshaw/startup/pull/1

Comment (spacerockzero) [reply bin/startup-start]: probably just making their debug function null-safe for use when debug is off

File: bin/startup-start [hunk 2]
```diff
@@ -201,9 +229,47 @@ cluster.on('disconnect', function(worker) {
  */
 
 cluster.on('listening', function(worker, address) {
+  var msg = "CLUSTER LISTENING";
+  if (worker && worker.id !== undefined) {
```
Comment (redbugz) [inline bin/startup-start]: is it possible for worker to be undefined here? if it's master there's a way to check for that

Comment (marybeshaw) [reply bin/startup-start]: again I am not sure because @gechols added it to my branch.

File: bin/startup-start (hunk 1 above)
Comment (spacerockzero) [reply bin/startup-start]: there are more obvious ways of doing that, but this works

Comment (redbugz) [reply bin/startup-start]: so why not use our standard `debug` library that handles of that for you?

Comment (marybeshaw) [reply bin/startup-start]: I can remove the debug stuff if you'd rather.

Comment (marybeshaw) [reply bin/startup-start]: We have have @gechols make a separate PR

File: bin/startup-start [hunk 3]
```diff
@@ -171,6 +183,11 @@ var args = [
 
 if (strict) args.unshift(strict);
 
+// pass unspecified argv values to node
+if (passAlongArgs) {
+  args.unshift.apply(args, passAlongArgs);
+}
```
Comment (redbugz) [inline bin/startup-start]: shame this has to be repeated three times. I'm assuming the underlying code is repetitive as well, so I guess we'll live with it until we refactor startup later to remove domains and improve error handling

Comment (marybeshaw) [reply bin/startup-start]: right, there are three modes: debug, development, and production. refactoring would be great but i was trying to touch the code as little as possible in this case.

**Comment by spacerockzero** [comment]
looks good

**Pull request merged**